mscr-ui/src/modules/form/index.tsx

```diff
@@ -543,7 +544,8 @@ export default function FormModal({
   }
 
   return (
-    <Modal
+    <>
```

Comment:
You don't need to add a fragment if you're returning just one element, like here everything is already inside the Modal element. It doesn't have any ill effect, but just reading the code it makes you expect a list of elements.